src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -822,11 +822,6 @@ public void CopyTo(int sourceIndex, char[] destination, int destinationIndex, in
                 throw new ArgumentNullException(nameof(destination));
             }
 
-            if (count < 0)
-            {
-                throw new ArgumentOutOfRangeException(nameof(count), SR.Arg_NegativeArgCount);
-            }
```

Comment:
Technically this a breaking change, as if multiple invalid inputs were provided, it'll change the order in which we check and thus will change what exceptions are thrown.  That said, I personally think that's an acceptable level of break, but others should weigh in... @weshaggard?